### DISCUSSION TITLE
Fancybox 4 / resize function for dynamic height

### DISCUSSION THREAD

**Discussion by novinet-dsteffen** [fancyapps/ui#339] [Q&A]
Hello,
I am using Fancybox 4 to load content within an iframe. Is there a resize or update function in case of changing dynamically (e.g. slideDown()) the content of the iframe?
Thanks, Daniel

**Comment by fancyapps** [comment]
Hi,

You could trigger auto-sizing like this:

```js
Fancybox.getInstance().Carousel.trigger("refresh");
```

**Comment by novinet-dsteffen** [comment]
Thank you. Unfortunatelly i get an error:
Uncaught TypeError: Cannot read properties of null (reading 'Carousel')

I changed it to "parent.Fancybox ..." Now i get a true, when I run it with console.log(), but still nothing happens, it does not refresh.

**Reply by fancyapps** [reply]
Can you create a demo? You could, for example, fork https://codesandbox.io/s/fancybox-iframe-k4i41 Because it should work, but maybe the script can not determine the height.

**Comment by novinet-dsteffen** [comment]
sorry, it helped ;-) just had to wait for the slideDown to be completed ... thx!!!

**Reply by fancyapps** [reply]
Ah, ok.
